src/main/java/org/folio/service/finance/transaction/EncumbranceService.java

```diff
@@ -384,36 +380,4 @@ protected void checkCustomTransactionError(Throwable fail) throws RuntimeExcepti
     }
   }
 
-  private Future<Void> unreleaseEncumbrancesFirst(EncumbrancesProcessingHolder holder, RequestContext requestContext) {
```

Comment:
great, reduced so many calls with this changes, that used in lot of flows
![image](https://github.com/folio-org/mod-orders/assets/25097693/80968eb2-8b12-4dcd-986d-98f45410dfb5)